pr-template.md

```diff
@@ -0,0 +1,31 @@
+# Pull Request Template <sup>1</sup>
+
+## Issue or Feature
+
+*Description:*
+
+*Branch Name:*
+
+### What has been changed?
+
+#### **Type of Change**
+
+- [ ] New feature
+- [ ] Bug fix
+- [ ] Updated documentation
+- [ ] Tests added
+- [ ] Other
+
+#### **Checklist** (check any applicable)
+
+- [ ] Comments included in Pull Request
+- [ ] Updated/new code was tested
+- [ ] Environmental variables added
+
+#### **Sources**
+
+*Please share any relevant sources or documentation*
+
+##### Footnotes
```

Comment:
And also this `Footnotes` section can be _replaced_ by a before and after screenshots section.

It would be great if you can change that😊